store/sqlstore/upgrade.go

```diff
@@ -328,8 +328,12 @@ func UpgradeDatabaseToVersion46(sqlStore SqlStore) {
 	//TODO: Uncomment folowing when version 4.6 is released
 	//if shouldPerformUpgrade(sqlStore, VERSION_4_5_0, VERSION_4_6_0) {
 
-	sqlStore.CreateColumnIfNotExists("IncomingWebhooks", "PostUsername", "varchar(64)", "varchar(64)", "")
-	sqlStore.CreateColumnIfNotExists("IncomingWebhooks", "PostIconURL", "varchar(1024)", "varchar(1024)", "")
+	//TODO: Remove these remove calls when version 4.6 is released. The columns were renamed.
+	sqlStore.RemoveColumnIfExists("IncomingWebhooks", "PostUsername")
```

Comment:
As discussed with @jwilander, this allows us to seamlessly migrate CI servers and such from the changes already committed. This shouldn't be part of the production release of this feature.